File: Readio/src/main/java/ninegle/Readio/mypage/mapper/PointMapper.java
package ninegle.Readio.mypage.mapper;

import org.springframework.stereotype.Component;

import ninegle.Readio.mypage.dto.response.PointResponseDto;

@Component
public class PointMapper {
	public PointResponseDto toDto(User user) {
		return new PointResponseDto(user.getPoint());
	}
}

File: Readio/src/main/java/ninegle/Readio/mypage/mapper/PointMapper.kt
package ninegle.Readio.mypage.mapper

import ninegle.Readio.mypage.dto.response.PointResponseDto
import ninegle.Readio.user.domain.User
import org.springframework.stereotype.Component
@Component
class PointMapper {
    fun toDto(user: User): PointResponseDto {
        return PointResponseDto(
            currentPoint = user.point
        )
    }
}